lmdb-sys/build.rs

```diff
@@ -48,38 +48,40 @@ fn main() {
     println!("cargo:rerun-if-changed=src/lib.rs");
     generate::generate();
 
-    let mut lmdb = PathBuf::from(&env::var("CARGO_MANIFEST_DIR").unwrap());
-    lmdb.push("lmdb");
-    lmdb.push("libraries");
-    lmdb.push("liblmdb");
-
     if cfg!(feature = "with-fuzzer") && cfg!(feature = "with-fuzzer-no-link") {
         warn!("Features `with-fuzzer` and `with-fuzzer-no-link` are mutually exclusive.");
         warn!("Building with `-fsanitize=fuzzer`.");
     }
 
-    if pkg_config::probe_library("lmdb").is_err() {
-        let mut builder = cc::Build::new();
+    if let Err(_) = std::env::var("LMDB_NO_BUILD") {
```

Comment:
I think the previous behavior is to find pkgconfig by default. I'd prefer it stay that way. Otherwise it is going to require modifications to all existing build that depends on pkgconfig.

You can fall back to building lmdb from source if pkg_config is not found. WDYT?

Reply:
We will still run pkg-config by default, one has to define the `LMDB_NO_BUILD` env variable explicitly to avoid running pkg-config.

Reply:
I see. I was reading it wrong.